styles/theme-base.css

```
@@ -555,8 +555,7 @@ div.classsynopsisinfo_comment {
 
 pre.info {
     border: 1px solid;
-    margin: 1rem 2rem 1.3rem;
-    margin-right: 0.8rem;
+    margin: 1rem 0.8rem 2rem 1.3rem;
```

```suggestion
    margin: 1rem 0.8rem 1.3rem 2rem;
```
You swapped bottom with left

---

Thanks for catching this embarrassing mistake. Fixed now.